src/tests/ftest/util/file_count_test_base.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import re
+from ior_test_base import IorTestBase
+from mdtest_test_base import MdtestBase
+from avocado.core.exceptions import TestFail
+
+
+def get_rf(oclass):
+    """Return redundancy factor based on the oclass.
+
+    Args:
+        oclass(string): object class.
+
+    return:
+        redundancy factor(int) from object type
+    """
+    rf = 0
+    if "EC" in oclass:
+        tmp = re.findall(r'\d+', oclass)
+        if tmp:
+            rf = int(tmp[1])
+    elif "RP" in oclass:
+        tmp = re.findall(r'\d+', oclass)
+        if tmp:
+            rf = int(tmp[0]) - 1
+    else:
+        rf = 0
+    return rf
+
+
+# pylint: disable=attribute-defined-outside-init
+class FileCountTestBase(MdtestBase, IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR and MDTEST to create large number of files.
+
+    :avocado: recursive
+    """
+
+    def add_containers(self, oclass=None):
+        """Create a list of containers that the various jobs use for storage.
+
+        Args:
+            pool: pool to create container
+            oclass: object class of container
+
+
+        """
+        # Create a container and add it to the overall list of containers
+        container = self.get_container(self.pool)
+        # don't include oclass in daos cont cmd; include rf based on the class
+        if oclass:
+            container.oclass.update(oclass)
+            redundancy_factor = get_rf(oclass)
+            rf = 'rf:{}'.format(str(redundancy_factor))
+        properties = container.properties.value
+        cont_properties = (",").join(filter(None, [properties, rf]))
+        if cont_properties is not None:
+            container.properties.update(cont_properties)
+        container.create()
+        return container
+
+    def run_file_count(self):
+        """Run the large file count test.
+
+        Args:
+            rc (bool, optional): If release candidate set to true. Defaults to False.
+        """
+        saved_container = []
```

Comment:
```suggestion
        saved_containers = []
```
It might make sense to make it plural.